src/api/admin.ts

```diff
@@ -1,30 +1,56 @@
 import { ServiceKey } from './types/admin'
-import { Domain } from './types/general'
+import { Domain, ProviderService, ServiceResponse } from './types/general'
 import { getAvailableDomains, setData, getProviderKeys } from './lib/data'
 import express from 'express'
 import uuid4 from 'uuid/v4'
 import util from 'util'
 import cp from 'child_process'
 import serviceConfig from './serviceConfig'
-import * as goDaddy from '../providers/goDaddy'
+import providers from '../providers'
 
 const app = express.Router()
 const exec = util.promisify(cp.exec)
 
 app.post('/sslCerts', async (req, res) => {
   const { service, selectedDomain } = req.body
+
+  const serviceResponse: ServiceResponse = {
+    success: true,
+    message: 'SSL Certs and domain name records successfully created'
+  }
   try {
     const serviceKeys = getProviderKeys().filter(d => d.service === service)
     const { keys } = serviceConfig[service]
     const envVars = keys.reduce((acc: string, key: string) => {
       const { value } = serviceKeys.find(d => d.key === key) || { value: '' }
       return acc + `${key}=${value} `
     }, '')
-    const { stdout, stderr } = await exec(
```

Comment:
If stdout is deleted here. How will expiration date, key and cert be obtained? Is line 40-50 the setup for information storage.